vizro-core/docs/pages/user_guides/navigation.md

```diff
@@ -0,0 +1,138 @@
+# How to use the navigation panel
+
+This guide shows you how to use and customize the left navigation panel in your dashboard.
+
+The [`Dashboard`][vizro.models.Dashboard] model also accepts a `navigation` argument, where you can enter the [`Navigation`][vizro.models.Navigation] model. A [`Navigation`][vizro.models.Navigation] in your dashboard facilitates grouping and navigating through your pages.
+
+## Using the default navigation
+
+By default, a navigation panel with an accordion item per page and a default title of `SELECT PAGE` is added to your dashboard. To leverage the navigation bar, you do not need to configure anything on top.
+
+!!! example "Default navigation"
+    === "app.py"
+        ```py
+        from vizro import Vizro
+        import vizro.plotly.express as px
+        import vizro.models as vm
+
+        iris = px.data.iris()
+
+        page_1 = vm.Page(
+            title="My first page",
+            components=[
+                vm.Graph(id="scatter_chart", figure=px.scatter(iris, x="sepal_length", y="petal_width", color="species")),
+            ],
+        )
+
+        dashboard = vm.Dashboard(pages=[page_1])
+
+        Vizro().build(dashboard).run()
+        ```
+    === "app.yaml"
+        ```yaml
+        # Still requires a .py to register data connector in Data Manager and parse yaml configuration - see from_yaml example
+        pages:
+          - components:
+              - figure:
+                  _target_: scatter
+                  data_frame: iris
+                  x: sepal_length
+                  y: petal_width
+                  color: species
+                id: scatter_chart
+                type: graph
+            title: My first page
+        - components:
+        ```
+    === "Result"
+        [![Dashboard]][Dashboard]
+
+    [Dashboard]: ../../assets/user_guides/navigation/default_navigation.png
+
+## Customizing your navigation panel
+
+If you want to deviate from the default title `SELECT PAGE` and instead provide a title for a group of selected pages, do the following steps:
+
+1. Provide a `dict` mapping of the page group title and a list of page IDs to the `pages` argument of the [`Navigation`][vizro.models.Navigation] model
+
+```
```

Comment:
These need to be indented to get the numbering of the enumeration correct. I tried to suggest that in the last review, but maybe the formatting messed up.

One good thing to do is to always check documentation rendered, there are two options to do so:
1. `hatch run docs:serve` and follow the local link
2. Check the RTD docs on the PR - this is also great as it shows the docs as they would be built finally